packages/common/src/store/user-list/index.ts

```diff
@@ -37,7 +37,3 @@ export * from './followers/types'
 export * as favoritesUserListActions from './favorites/actions'
 export * as favoritesUserListSelectors from './favorites/selectors'
 export * from './favorites/types'
-
-export * as searchUserListActions from './search/actions'
```

Comment:
In theory, I guess I could keep the same store structure as though it were a userList and have searchUsersModal use those actions/sagas. I made my own but what do you think @dylanjeffers ?